tests/test_directives.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2017 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import pytest
+import sys
+from copy import deepcopy
+
+from flow.directives import Directives
+from flow.directives import NP, NRANKS, NGPU, EXECUTABLE, OMP_NUM_THREADS
+
+
+@pytest.fixture()
+def setUp(test_class=Directives, **kwargs):
+    def _setup(test_class=Directives, **kwargs):
+        return test_class(**kwargs)
+    return _setup
+
+
+available_directives_list = [NP, NRANKS, NGPU, EXECUTABLE, OMP_NUM_THREADS]
```

Comment:
This would make a good fixture.

Reply:
Noted

Reply:
This is marked as resolve, but there is still no fixture.